CONTRIBUTING.md

```diff
@@ -190,17 +192,17 @@ Then execute `npm run debug`.
 - Click **Load Temporary Add-on** button
 - Open the `build/debug/firefox/manifest.json` file.
 
-If you execute `npm run debug:watch` instead of `npm run debug`.
-It will after making any code changes, the project will be automatically recompiled.  
+If you execute `npm run debug:watch` instead of `npm run debug`, the project will be automatically recompiled
+after making any code changes.
 
 ## Tips
 
-For editing the code you can use any text editor or web IDE (like [Visual Studio Code](https://code.visualstudio.com), [Atom](https://atom.io/), or [WebStorm](https://www.jetbrains.com/webstorm/)).
+For editing the code, you can use any text editor or web IDE (like [Visual Studio Code](https://code.visualstudio.com), [Atom](https://atom.io/), or [WebStorm](https://www.jetbrains.com/webstorm/)).
 
 **Please preserve code style** (whitespaces etc).
 This can automatically be done by executing `npm run code-style`.
 
-Run tests by executing `npm test` to make sure it will pass the test.
+Run tests by executing `npm test` to make sure your code will pass.
```

Comment:
Pass what? 😄

Reply:
The *tests* (second word).  I can add "them" to the end of the sentence if you like.  What's the best way to edit what I've already issued a PR for?  When you make a suggestion, I can just accept it quickly (if I like it 😸 ), but I'm not seeing a quick way for me to just add a word to the PR without finding the file in my fork.

Reply:
```suggestion
Run tests by executing `npm test` to make sure your code will pass them.
```

Reply:
> What's the best way to edit what I've already issued a PR for?

File changed tab -> 3 dots on the first file -> Edit file.